org.lflang/src/org/lflang/GenerateLinguaFranca.mwe2

```diff
@@ -44,6 +44,7 @@ Workflow {
                 //composedCheck = "org.eclipse.xtext.validation.ImportUriValidator"
                 // Generates checks for @Deprecated grammar annotations, an IssueProvider and a corresponding PropertyPage
                 generateDeprecationValidation = true
+                generateXtendStub = true
```

Comment:
I couldn't find an Xtend documentation for this (as expected). What does this accomplish?

Reply:
It tells xtext to generate LFValidator.xtext rather than LFvalidator.java